### DISCUSSION TITLE
Incorrect school identified based on school email address.

### DISCUSSION THREAD

**Discussion by Babyyoda777** [community/community#60618] [GitHub Education]
### Hi everyone,

Hi, I have the school email @qerdp.co.uk and it is incorrectly identifying this as being from Queen Elizabeth's Grammar School Ashbourne Academy where as it is actually meant to be just Queen Elizabeth's School, Barnet (https://www.qebarnet.co.uk). Students at our school do not have an @qebarnet.co.uk email but have been designated the @qerdp.co.uk email only. I am unable to verify due to this issue as my documentation is rejected everytime (I have tried 11 times). This is only an issue with the new system as I successfully verified with the old one last year. Please fix asap, horrible system.

**Comment by Babyyoda777** [comment]
![Screenshot 2023-07-11 at 17 53 54](https://github.com/community/community/assets/80623330/6c610f16-d855-447e-8dbb-0ed845b8d48f)
Additionally according to GitHub it requires these emails but these emails are not linked in anyway to our school.

**Comment by BenjaminDecaillet** [comment]
Hello,

Same issue here with Unidistance Switzerland:
![Re-submit_Github](https://github.com/community/community/assets/26813570/7a0b78cf-65dd-4aec-ab0d-f37629f78f9b)

**Comment by pinkyplant** [comment]
Same issue here, school uses a student prefix for students and normal domain for teachers

**Comment by samus-aran** [comment]
Hi @Babyyoda777  👋

Your school info has since been updated and we can confirm you have been successful in your application. Thank you for your patience and enjoy yourself on your GitHub Education journey.

For everyone else if you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

You should provide evidence to support if your school has additional academic domains, has changed its academic email issuing policy, or does not provide academic emails. Reach out to your institution's IT team for this. 

Hope this helps! I'll be closing this discussion due to lack of recent activity.
